src/index.js

```diff
@@ -0,0 +1,77 @@
+import './style.css';
+
+const input = document.querySelector('#new-task-input');
+const submitButton = document.querySelector('.add');
+const taskList = document.querySelector('#tasks');
+let tasks = [];
+
+if (localStorage.getItem('tasks')) {
+  tasks = JSON.parse(localStorage.getItem('tasks'));
+}
+
+/*eslint-disable*/
```

Comment:
- [ ] Please avoid disabling linter rules as they ensure you follow a consistent code format and also apply best practices, instead consider fixing the errors! 🤓 🔧